cg/store/api/add.py

```diff
@@ -141,6 +141,7 @@ def add_sample(
         received: dt.datetime = None,
         original_ticket: str = None,
         tumour: bool = False,
+        delivered_at: dt.datetime = None,
```

Comment:
Does this make sense? Should one expect to be able to create a sample object that has already been deleted?

Reply:
I see your point. This could indeed be handled with the **kwargs argument in that case-
The delivered_at addition is mainly used to test the query.

Reply:
We generally want to avoid using kwargs, since it reduces readability. And changing production code for tests to work is a bit sketchy. I suggets creating a sample in the test, and then changing the `delivered_at` field there.

Reply:
In regard to the **kwargs we can think if taking it away but maybe it a separate PR focussing on only that.

Reply:
I think we need to keep it in some places, but avoid relying on it or introducing it, when there are simpler ways to solve it 😊